src/sphinx/formats/universal.rst

```diff
@@ -158,6 +158,12 @@ If you want to force local for the `tgz` output add this line:
   universalArchiveOptions in (Universal, packageZipTarball) := Seq("--force-local", "-pcvf")
 
 This will set the cli options for the `packageZipTarball` task in the `Universal` plugin to use the options ``--force-local`` and ``pcvf``.
+Be aware that the above line will overwrite the default options.  You may want to prepend your options, doing something like:
+
+.. code-block:: scala
+
+  universalArchiveOptions in (Universal, packageZipTarball) :=
```

Comment:
Thanks for clarifying this. It would be nicer add this as a second example as this is more a sbt operator confusion then a native-packager documentation issue. The `:=` operator overrides an existing setting whereas `+=` / `++=` appends something to an existing setting that is a `Seq`. So you change the example to

```scala
// override settings
universalArchiveOptions in (Universal, packageZipTarball) := Seq("--force-local", "-pvcf")
// addditional settings
universalArchiveOptions in (Universal, packageZipTarball) ++= Seq("--exclude", "*~"
```